doc/source/train/distributed-pytorch/experiment-tracking-callbacks.rst

```diff
@@ -0,0 +1,86 @@
+.. _train-monitoring:
+
+Experiment tracking and Callbacks
```

Comment:
I personally feel like for Experiment Tracking, we should lean towards the native experiment tracking framework APIs that the user is most familiar with, rather than have them use Callbacks.

That said, I'm not sure how we want to position/expose Callbacks right now - maybe primarily for early stopping?

Reply:
Sounds good. I just want to call out that I think structure-wise, we should move Callbacks so they're not a first-class concept for experiment tracking in the page title. As a note I think Tune docs do a pretty good job here.

As a straw-man, we can potentially have:
1. A user guide for generic Callback definition/usage, similar to [Tune].(https://docs.ray.io/en/master/tune/tutorials/tune-metrics.html)
2. A reference from the Monitoring Metrics user guide.
3. A reference from the Experiment Management user guide.

Also cc @xwjiang2010 regarding experiment tracking.